### DISCUSSION TITLE
Will we ever want a third release channel?

### DISCUSSION THREAD

**Discussion by robscott** [kubernetes-sigs/gateway-api#1172] [Polls]
Currently we've proposed releasing v0.5.0 with [two release channels](https://gateway-api.sigs.k8s.io/concepts/versioning/#release-channels-eg-experimental-stable) - experimental and stable. The stable release channel will only include resources that have graduated to beta. In the future, when we graduate some resources to GA, will we want a third release channel that only includes resources that have graduated to GA? 

If so, what would we name that? We could potentially add a "GA" channel on top of our existing channels to result in the following names:

```
Experimental -> Stable -> GA
```

Alternatively, we could save the term "stable" to refer to resources that have graduated to GA, and introduce a new intermediate term, such as "standard" (other alternatives include "common", "conventional", or "regular").

```
Experimental -> Standard -> Stable
```

I think we will likely be able to go a long way, potentially forever, with only two release channels. With that said, it seems like it would be good to leave room to add a third release channel in the future.

**Comment by youngnick** [comment]
I'm a big +1 to going with "standard" for this release, and saving "stable" for later, now that you mention it.
